liquibase-standard/src/main/java/liquibase/diff/output/changelog/core/MissingIndexChangeGenerator.java

```diff
@@ -70,6 +70,12 @@ public Change[] fixMissing(DatabaseObject missingObject, DiffOutputControl contr
             change.addColumn(new AddColumnConfig(column));
         }
 
+        for (String column : index.getIncludedColumns()) {
```

Comment:
I'm thinking we should surround this piece as well inside of `IF` block for an `MSSQL` DB , because otherwise when running `Snapshot`/`Generate-Changelog` commands missing indexes could be added in the results for other DBs other than `MSSQL`.